sdcm/cluster.py

```diff
@@ -2226,6 +2226,11 @@ def get_test_keyspaces(self):
         out = self.run_cqlsh(node=self.nodes[0], cql_cmd='select keyspace_name from system_schema.keyspaces')
         return [ks.strip() for ks in out.split('\n')[3:-3] if 'system' not in ks]
 
+    def get_keyspace_tables(self, keyspace):
```

Comment:
There is already something like this called get_non_system_ks_cf_list

Reply:
Changed to use this function